packages/runtime/container-runtime/src/test/documentSchema.spec.ts

```diff
@@ -46,9 +59,11 @@ describe("Runtime", () => {
 		return new DocumentsSchemaController(
 			true, // existing,
 			0, // snapshotSequenceNumber
-			config as IDocumentSchemaCurrent, // old schema,
+			config as IDocumentSchema, // old schema,
```

Comment:
Is this correct/needed?

Reply:
Jason had the same comment here - https://github.com/microsoft/FluidFramework/pull/24685#discussion_r2109745248